comictaggerlib/main.py

```diff
@@ -28,6 +28,7 @@
 from collections.abc import Collection
 from typing import cast
 
+import plugin_update_manager
```

Comment:
This module isn't in this commit yet. Did you have something for this or was that next on your list?

Reply:
I may have forgot to `add` it...

I saw the comment about using YAML etc. for manifest but I just included what was supposed to be with the initial push.

And some mypy complaints to fix.